pyproject.toml

```diff
@@ -1,10 +1,193 @@
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Natural Language :: English",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: GIS",
+    "Topic :: Scientific/Engineering :: Hydrology",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dynamic = [
+    "readme",
+    "version",
+]
+dependencies = [
+    "aiohttp",
+    "cartopy",
+    "cdo",
+    "cdsapi",
+    "cf-units",
+    "cfgrib",
+    "cftime",
+    "cmocean",
+    "dask!=2024.8.0",  # https://github.com/dask/dask/issues/11296
+    "distributed",
+    "ecmwf-api-client",
+    "eofs",
+    "ESMPy",  # not on PyPI
+    "esmvalcore",
+    "esmf-regrid>=0.10.0",  # iris-esmf-regrid #342
+    "fiona",
+    "fire",
+    "fsspec",
+    "GDAL",
+    "ipython<9.0",  # github.com/ESMValGroup/ESMValCore/issues/2680
+    "jinja2",
+    "joblib",
+    "lime",
+    "mapgenerator>=1.0.5",
+    "matplotlib",
+    "natsort",
+    "nc-time-axis",
+    "netCDF4",
+    "numba",
+    "numpy!=1.24.3",  # severe masking bug
+    "openpyxl",
+    "packaging",
+    "pandas",
+    "progressbar2",
+    "pyproj>=2.1",
+    "pys2index",
+    "python-dateutil",
+    "pyyaml",
+    "rasterio>=1.3.10",
+    "requests",
+    "ruamel.yaml",
+    "scikit-image",
+    "scikit-learn>=1.4.0",  # github.com/ESMValGroup/ESMValTool/issues/3504
+    "scipy",
+    "scitools-iris>=3.11",
+    "seaborn",
+    "seawater",
+    "shapely>=2",
+    "xarray>=0.12.0",
+    "xesmf>=0.7.1",
+    "xgboost>1.6.1",  # github.com/ESMValGroup/ESMValTool/issues/2779
+    "xlsxwriter",
+    "zarr",
+]
+description = "A community tool for pre-processing data from Earth system models in CMIP and running analysis scripts"
+license = {text = "Apache License, Version 2.0"}
+name = "ESMValTool"
+requires-python = ">=3.9"
+
+[project.optional-dependencies]
+# Execute `pip install .[test]` once and then use `pytest` to run tests
+test = [
+    "pre-commit",
+    "pytest>6.0.0",
+    "pytest-cov>=2.10.1",
+    "pytest-env",
+    "pytest-html!=2.1.0",
+    "pytest-metadata>=1.5.1",
+    "pytest-mock",
+    "pytest-xdist",
+]
+doc = [
+    "autodocsumm>=0.2.2",
+    "nbsphinx",
+    "sphinx>=6.1.3",
+    "pydata_sphinx_theme",
+]
+develop = [
+    "esmvaltool[test,doc]",
+    "codespell",
+    "docformatter",
+    "imagehash",
+    "pre-commit",
+    "prospector[with_pyroma]>=1.12",
+    "vprof",
+    "yamllint",
+]
+
+[project.scripts]
+esmvaltool = "esmvalcore._main:run"
```

Comment:
This looks like something from the ESMValCore package, should it be here?

Reply:
no buds, you can't take that out, that'll result in `esmvaltool` executable not being found anymore

Reply:
that comes only when ESMValCore is installed from source, not when it's installed off conda as a dependency

Reply:
If I run `conda create -n test esmvalcore`; `conda activate test`; `which esmvaltool`, I get `/home/bandela/mambaforge/envs/test/bin/esmvaltool`, as expected.